index.html

```diff
@@ -717,11 +717,87 @@
 					</section>
 					<section>
 						<h1>cURL</h1>
-						<p>TODO: cURL intro, exercise</p>
+						<p><span class="fragment">&quot;Command line tool and library for transferring data with URLs&quot;</span></p>
+						<p><span class="fragment"><a href="https://curl.haxx.se/download.html" target="_blank">https://curl.haxx.se/download.html</a></span></p>
+					</section>
+					<section>
+<h2>Exercise 13: cURL</h2>
+<pre><code class="bash">
+# Get your own profile
+curl https://api.github.com/users/&lt;username&gt;
+
+# Same, but authenticated and show some information:
+curl -i -u &lt;username&gt; https://api.github.com/user
+
+# Get data for a repo
+curl -i -u &lt;username&gt; https://api.github.com/repos/hbz/swib16-workshop
+
+</code></pre>
 					</section>
 					<section>
 						<h1>jq</h1>
-						<p>TODO: jq intro, exercise</p>
+						<p><span class="fragment">Last cURL output was pretty long, we often want specific values</span></p>
+						<p><span class="fragment">jq: &quot;A lightweight and flexible command-line JSON processor&quot;</span></p>
+						<p><span class="fragment"><a href="https://stedolan.github.io/jq/" target="_blank">https://stedolan.github.io/jq/</a></span></p>
+						<p><span class="fragment"><a href="https://jqplay.org/">https://jqplay.org/</a></span></p>
+					</section>
+					<section>
+<h2>Exercise 14: jq basics</h2>
+<pre><code class="bash"># We start with cURL again:
+curl 'https://api.github.com/repos/hbz/swib16-workshop/commits'
+
+# We pipe the JSON to the jq format
+curl 'https://api.github.com/repos/hbz/swib16-workshop/commits' | jq
```

Comment:
While `curl 'https://api.github.com/repos/hbz/swib16-workshop/commits'` is fine, this currenrtly doesn't work for me. Output:

```
$ curl 'https://api.github.com/repos/hbz/swib16-workshop/commits' | jq
  % Total    % Received % X
jq - commandline JSON processor [version 1.3]
ferUsage: jq [options] <jq filter> [file...]

d  AvFor a description of the command line options and
ehow to write jq filters (and why you might want to)
rsee the jq manpage, or the online documentation at
ahttp://stedolan.github.com/jq

ge Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  4  103k    4  4360    0     0   5689      0  0:00:18 --:--:--  0:00:18  5684
curl: (23) Failed writing body (1106 != 1370)
```